modules/node_modules/@frogpond/ccc-rss-feed/index.js

```diff
@@ -42,13 +42,23 @@ export function convertRssItemToStory(item) {
 		: content.substr(0, 250)
 	excerpt = JSDOM.fragment(excerpt).textContent.trim()
 
+	let featuredImage = null
+	if (item.querySelector('enclosure')) {
+		let featuredMediaInfo = item.querySelector('enclosure')
+		let containsImage = featuredMediaInfo.getAttribute('type').includes('image')
```

Comment:
For future reference, `enclosure` [_is_ part of the RSS 2.0 spec](https://www.rssboard.org/rss-draft-1#element-channel-item-enclosure).  `featuredImage` is specifically for `image`s, but if an RSS feed is fancy and embeds videos or something, we'd get a different MIME type and not include it. (which is perfectly fine)

One nit, though: this `image` check isn't position-dependent. If, for example, my `enclosure` had a MIME-type of `application/vnd.oci.image.manifest.v1+json`, this would match. Not sure what the exposure from that is, but maybe do `.startsWith('image')` instead?

Reply:
I had a concern about the current implementation too, so your comment cements that this needs to change. I'll address the "starts with" part!